src/components/ViewFieldIngelogdExtraInfo/ViewFieldIngelogdExtraInfo.js

```diff
@@ -102,12 +113,26 @@ const ViewFieldIngelogdExtraInfo = ({ crudObject, hideEdit, className }) => {
     )
 }
 
+/**
+ * Component that renders the EigenarenList component, which displays the ownerlist.
+ *
+ * @component
+ *
+ * @param {Array} eigenaren - Parameter containing a collection of owners.
+ */
 const EigenarenList = ({ eigenaren }) => {
     const getUsername = (item) => {
         if (!item) return null
         return item.Gebruikersnaam
     }
 
+    /**
+     * Function that gets the abbrevation from the username.
+     *
+     * @function
+     *
+     * @param {Array} item - parameter containing a collection of users.
```

Comment:
type string

Reply:
I've changed the parameter description and type to string.